app/components/componentlibrary/FontStyles.js

```diff
@@ -0,0 +1,235 @@
+import React from 'react';
```

Comment:
I wasn't sure how separated we would want the component library and if/how we might want to eventually re-work existing components. If you're okay with using and elements directory within components, rather than having the library split out that works for me, as simple as we can keep the paths the better.